.github/workflows/main.yml

```diff
@@ -11,7 +11,7 @@ jobs:
     name: Tests pandas / files
     runs-on: ${{ matrix.os }}
     env:
-      BACKENDS: "pandas csv parquet hdf5"
+      BACKENDS: "pandas dask csv parquet hdf5"
```

Comment:
let's add this as a separate GH (basically copy the pandas section to a new Tests_dask)